lib/extensions/acts_as_helpers/active_record/base.rb

```diff
@@ -19,4 +29,10 @@ def manually_awarded?
       false
     end
   end
+
+  module LessonPlanItemClassMethods
+    def self.has_todo?
```

Comment:
@weiqingtoh Instead requiring users declare the class method. how something about `acts_as_lesson_plan_item, with_todo: true`